src/com/rcv/Tabulator.java

```diff
@@ -329,26 +329,26 @@ private boolean dropUWI(
   // returns: true if any candidates were eliminated
   private boolean dropCandidatesBelowThreshold(
     List<String> eliminated,
-    SortedMap<Float, LinkedList<String>> currentRoundTallyToCandidates
+    SortedMap<BigDecimal, LinkedList<String>> currentRoundTallyToCandidates
   ) {
     if (
-      eliminated.isEmpty() &&
-      config.minimumVoteThreshold() != null &&
-      currentRoundTallyToCandidates.firstKey() < config.minimumVoteThreshold()
+      eliminated.isEmpty() && // <-- TODO: Louis should this condition be here?
+      config.minimumVoteThreshold().signum() == 1 &&
+      currentRoundTallyToCandidates.firstKey().compareTo(config.minimumVoteThreshold()) == -1
     ) {
       // tally indexes over all tallies in the current round
-      for (Float tally : currentRoundTallyToCandidates.keySet()) {
-        if (tally < config.minimumVoteThreshold()) {
+      for (BigDecimal tally : currentRoundTallyToCandidates.keySet()) {
+        if (tally.compareTo(config.minimumVoteThreshold()) == -1) {
           // candidate indexes over all candidates who received this tally
           for (String candidate : currentRoundTallyToCandidates.get(tally)) {
             eliminated.add(candidate);
             log(
-              "Eliminated %s in round %d because they only had %f vote(s), below the minimum " +
-                "threshold of %d.",
+              "Eliminated %s in round %d because they only had %s vote(s), below the minimum " +
+                "threshold of %s.",
               candidate,
               currentRound,
-              tally,
-              config.minimumVoteThreshold()
+              tally.toString(),
+              config.minimumVoteThreshold().toString()
             );
           }
         }
```

Comment:
micro-optimization: since we know that `currentRoundTallyToCandidates` is sorted, we can add an `else break` here

Reply:
done!